build.proj

```diff
@@ -2,8 +2,14 @@
 <Project ToolsVersion="12.0" DefaultTargets="BuildAndTest" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="dir.props" />
 
+  <!-- required to build the projects in their specified order -->
+  <PropertyGroup>
+    <SerializeProjects>true</SerializeProjects>
+  </PropertyGroup>
+
   <ItemGroup>
     <Project Include="src\dirs.proj" />
+    <Project Include="src\post.proj" />
```

Comment:
Not really super important, but I like being able to run msbuild on a directory and have it select the most obvious thing that I want to build (in this case the product binaries under the current directory). I think it will complain about there being more than one project file with this.